codingStandards/CodingStandard-Java.md

```diff
@@ -602,7 +634,7 @@ if (isDone) doCleanup();</pre>
 
 This is for debugging purposes. When writing on a single line, it is not apparent whether the test is really true or not.
 
-**11. Single statement conditionals should still be wrapped by curly brackets**
+**12. Single statement conditionals should still be wrapped by curly brackets**
```

Comment:
@damithc Shall we add a full stop to these? Since we're already modifying the lines and it's cosmetic.